docker-compose.test.yml

```diff
@@ -56,7 +57,7 @@ services:
           mknod /dev/net/tun c 10 200
         fi
 
-        /command/with-contenv npm run test-unit
+        # /command/with-contenv npm run test-unit
```

Comment:
Why are we skipping this test?

Reply:
It was a local change that wasn't meant to be included, but basically when I was debugging the second test command I didn't want to wait ages for this one each time